### PR TITLE
refactor folder structure of static FAQs

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
I couldn't help but to change it. IMHO it should clean the structure of folder a little bit 🙏 <br/><br/><br/><url>LiveURL: https://smartfaq-refactor-faq-folder-structure.surge.sh</url>

### CODE REVIEW

File: src/common/layout/ContentPage.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Switch, Route } from 'react-router-dom';
 import { Desktop } from '../Responsive';
 import Header from './Header';
 import StaticFAQ from '../../StaticFAQ';
-import FAQArticleDetail from '../../StaticFAQ/FAQArticleDetail';
+import FAQArticleDetail from '../../StaticFAQ/ArticleDetail/FAQArticleDetail';
```
Comment (kantord) [inline src/common/layout/ContentPage.js]: `ArticleDetail/FAQArticleDetail` sounds a bit too redundant. Also we have `FAQ` twice in the path.

Comment (kantord) [reply src/common/layout/ContentPage.js]: So maybe renaming `FAQArticleDetail` would remove both redundancies

Comment (jaroslav-kubicek) [reply src/common/layout/ContentPage.js]: fixed

File: src/StaticFAQ/FAQCategoryList.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ import BaggageInfo from './FAQExtraInfo/BaggageInfo';
 import BoardingPassesInfo from './FAQExtraInfo/BoardingPassesInfo';
 import FAQArticle from './FAQArticle';
 import FAQCategory from './FAQCategory';
-import Breadcrumbs from './Breadcrumbs';
+import Breadcrumbs from './breadcrumbs/Breadcrumbs';
```
Comment (kantord) [inline src/StaticFAQ/FAQCategoryList.js]: Maybe use `index.js` here? :stuck_out_tongue_closed_eyes: I know we don't really like it, but this looks redundant. Normally I probably wouldn't notice this, but this is a refactoring PR.

Comment (jaroslav-kubicek) [reply src/StaticFAQ/FAQCategoryList.js]: it's not possible due to relay, that's why we usually don't like `index.js` files

Comment (kantord) [reply src/StaticFAQ/FAQCategoryList.js]: ah, I didn't realize that! You're right. Thanks

**Pull request merged**